webrtc/simplecall.html

```diff
@@ -108,9 +108,8 @@
 
   // This function starts the test.
   test.step(function() {
-    navigator.getUserMedia({ video: true, audio: true },
-        getUserMediaOkCallback,
-        failed('getUserMedia'));
+    navigator.mediaDevices.getUserMedia({ video: true, audio: true })
+      .then(getUserMediaOkCallback, failed('getUserMedia'));
```

Comment:
The latter example of try/catching everything also ignores that synchronous functions can throw as well. IMHO try/catching every call is swimming upstream of the exception pattern.